js/code-docs.js

```diff
@@ -267,6 +267,40 @@ const App = () => {
   dangerouslySetInnerHTML2: `const App = () => {
   return <div innerHTML={{ __dangerousHtml: html }} />;
 }`,
+withStyles: `import { render } from 'nano-jsx/lib/core'
+import { withStyles } from 'nano-jsx/lib/withStyles'
+import { Fragment } from 'nano-jsx/lib/fragment'
+
+const App = () => {
+  // define CSS-Styles inside a Component
+	const css = \`
```

Comment:
Indent should only  be 2 spaces.

Reply:
Strange. I run prettier before pushing and in VS Code the indent had 2 spaces for me. I fixed it manually now and run prettier again. Should be good now.